plugins/metric_links/data/database.sql

```diff
@@ -1,22 +0,0 @@
-SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
-SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
-SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
-
-
--- -----------------------------------------------------
--- Table `metric_links_status`
--- -----------------------------------------------------
-CREATE TABLE IF NOT EXISTS `metric_links_status` (
```

Comment:
It currently doesn't.  I was planning to remove it manually, but perhaps an automated removal would be nice.